.github/workflows/ci_cron_weekly.yml

```diff
@@ -55,19 +55,14 @@ jobs:
             python: '3.11'
             toxenv: py311-test-devdeps-noscipy
 
-          - name: Python 3.12 with dev version of infrastructure dependencies
+          - name: Python 3.13 with dev version of infrastructure dependencies
             os: ubuntu-latest
-            python: '3.12'
+            python: '3.13'
             toxenv: py312-test-devinfra
 
-          - name: Python 3.13-dev with dev versions of key dependencies
-            os: ubuntu-latest
-            python: '3.13-dev'
-            toxenv: py313-test-devdeps
-
-          - name: Python 3.12 on macOS (x86_64) with all optional dependencies
+          - name: Python 3.13 on macOS (x86_64) with all optional dependencies
```

Comment:
Python 3.13 with all deps so early in release might not fly...

Reply:
yup. Just trying out my luck here. Will revert if (when ?) it comes down crashing into flames.

Reply:
nevermind I just did it in https://github.com/astropy/astropy/issues/17140